data/data/aws/vpc/sg-master.tf

```diff
@@ -253,3 +313,33 @@ resource "aws_security_group_rule" "master_ingress_services_udp" {
   self      = true
 }
 
+# For our AWS IPv6 environment, we run CoreDNS with host networking,
+# because it must use IPv4 to reach AWS DNS, so it can't be on our
+# IPv6 only SDN.
```

Comment:
TODO(@abhinavdahiya): get clarity on this

Reply:
Are you asking me for clarity?

This issue affects several pods.  They try to reach the IPv4-only AWS APIs and can't do it from the IPv6-only SDN.  We're also looking adding a secondary interface with multus that provides IPv4 connectivity.  For now, we have a couple of pods using host networking.

Reply:
https://github.com/openshift/enhancements/pull/87#issuecomment-562665545

Reply:
I added more details about this in the enhancement and the docs in this PR.